v2/controllers/marketplace/meterbase_controller.go

```diff
@@ -681,14 +861,17 @@ func (r *MeterBaseReconciler) installMetricStateDeployment(
 ) []ClientAction {
 	deployment := &appsv1.Deployment{}
 	service := &corev1.Service{}
-	serviceMonitor := &monitoringv1.ServiceMonitor{}
+	serviceMonitor0 := &monitoringv1.ServiceMonitor{}
```

Comment:
Can we enhance this name a bit? serviceMonitorShortName i.e. like serviceMonitorMetricState

Reply:
changed to be in step with the factory name